packages/lib-classifier/src/hooks/useFreehandLineReductions.js

```diff
@@ -0,0 +1,51 @@
+import { useStores } from '@hooks'
+import { useCaesarReductions } from './'
+
+export default function useFreehandLineReductions() {
+  const {
+    subjectViewer: {
+      frame
+    },
+    workflows: {
+      active: workflow
+    },
+    workflowSteps: {
+      active: step,
+      findTasksByType
+    }
+  } = useStores()
+
+  // NOTE: useFreehandLineReductions() and useTranscriptionReductions() are called regardless of reducer key
+  if (workflow.caesarReducer !== 'machine-learnt') return
```

Comment:
This returns `undefined` but your React component expects an object back from this hook.

Reply:
You can't have a conditional return before a hook, either. See [the rules of hooks](https://react.dev/warnings/invalid-hook-call-warning).